src/vivarium_gates_mncnh/components/pregnancy.py

```diff
@@ -55,17 +59,21 @@ def setup(self, builder: Builder):
         self.time_step = builder.time.step_size()
         self.randomness = builder.randomness.get_stream(self.name)
         self.birth_outcome_probabilities = builder.value.register_value_producer(
-            "birth_outcome_probabilities",
+            PIPELINES.BIRTH_OUTCOME_PROBABILITIES,
             source=self.lookup_tables["birth_outcome_probabilities"],
-            requires_columns=get_lookup_columns(
+            required_resources=get_lookup_columns(
                 [self.lookup_tables["birth_outcome_probabilities"]]
             ),
         )
         self.pregnancy_durations = builder.value.register_value_producer(
             PIPELINES.PREGNANCY_DURATION,
             self.get_pregnancy_durations,
             self,
-            requires_columns=[COLUMNS.PREGNANCY_OUTCOME],
+            required_resources=[
+                COLUMNS.PREGNANCY_OUTCOME,
+                COLUMNS.PARTIAL_TERM_PREGNANCY_DURATION,
+                COLUMNS.GESTATIONAL_AGE_EXPOSURE,
```

Comment:
@rmudambi I have to use the column instead of the birth exposure pipeline because the pipeline does change when we age the children.